### PR TITLE
adding Javascipt week 2 homework file

### PR DESCRIPTION
**Pull request by varsha84**

### CODE REVIEW

File: javascript/javascript1/week2/Candy-helper.js [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+
+function canBuyMoreCandy(amountToSpend)
+{
+    let totalAmount = 0;
+    for(let i = 0; i < boughtCandyPrices.length; i++)
+    {
+        totalAmount = totalAmount + boughtCandyPrices[i]
+    }
+
+    const results = (totalAmount < amountToSpend ? true : false);
```
Comment (FrederikNygart) [inline javascript/javascript1/week2/Candy-helper.js]: is the ternary operator needed?

File: javascript/javascript1/week2/Event-app.js [hunk 2]
```diff
@@ -0,0 +1,12 @@
+// find event day after some days later
+
+ days = ['Sunday', 'Monday', 'Tuesday', 'Wednesday', 'Thursday', 'Friday', 'Saturday'];
+
+function getEventWeekday(eventDays)
+{
+    const todayDate = new Date();
+    let eventDayOfweek = (todayDate.getDay() + eventDays) % 7;
+    return days[eventDayOfweek];
+}
```
Comment (FrederikNygart) [inline javascript/javascript1/week2/Event-app.js:10]: nice! 💯

File: javascript/javascript1/week2/flight-Booking.js [hunk 3]
```diff
@@ -0,0 +1,28 @@
+
+//Flight booking fullname function and Formal fullname
+
+function getFullname(firstname, surname, useFormalName)
```
Comment (FrederikNygart) [inline javascript/javascript1/week2/flight-Booking.js]: what if the person is a woman?

File: javascript/javascript1/week2/student-manager.js [hunk 4]
```diff
@@ -0,0 +1,48 @@
+//Student manager
+const class07Students = [];
+function addStudentToClass(studentName)
+{
+    if(studentName === "")
```
Comment (FrederikNygart) [inline javascript/javascript1/week2/student-manager.js]: what if the function is called like this `addStudentToClass(null)` or `addStudentToClass()`?

File: javascript/javascript1/week2/student-manager.js [hunk 5]
```diff
@@ -0,0 +1,48 @@
...
+        console.log("please provide student name");
+    }
+    else if(studentName === "Queen Margrethe")
+    {
+        class07Students.push(studentName);
+        console.log(studentName + " has been added to class" )
+    }
+    else if(class07Students.length < 6)
+    {
+            if(class07Students.includes(studentName))
```
Comment (FrederikNygart) [inline javascript/javascript1/week2/student-manager.js]: does this need to be nested?

File: javascript/javascript1/week2/student-manager.js [hunk 6]
```diff
@@ -0,0 +1,48 @@
...
+    if(studentName === "")
+    {
+        console.log("please provide student name");
+    }
+    else if(studentName === "Queen Margrethe")
+    {
+        class07Students.push(studentName);
+        console.log(studentName + " has been added to class" )
+    }
+    else if(class07Students.length < 6)
```
Comment (FrederikNygart) [inline javascript/javascript1/week2/student-manager.js]: this only allows 5 people in the class, the max amount is 6.

File: javascript/javascript1/week2/weather-wear.js [hunk 7]
```diff
@@ -0,0 +1,37 @@
...
+    {
+        return "wear shorts, skirt";
+
+    }
+    else if(temprature <= 35)
+    {
+        return "wear beach clothes"
+
+    }
+    else
```
Comment (FrederikNygart) [inline javascript/javascript1/week2/weather-wear.js]: redundant else block

**Review by FrederikNygart — CHANGES_REQUESTED** [review]
Good job! almost perfect 👍 contact me on slack if you have questions regarding my comments.

File: javascript/javascript1/week2/Candy-helper.js (hunk 1 above)
Comment (varsha84) [reply javascript/javascript1/week2/Candy-helper.js:40]: Its not needed. I fixed it

Comment (varsha84) [reply javascript/javascript1/week2/Candy-helper.js]: it is not needed. I fixed it

File: javascript/javascript1/week2/flight-Booking.js (hunk 3 above)
Comment (varsha84) [reply javascript/javascript1/week2/flight-Booking.js]: I fixed it also.

File: javascript/javascript1/week2/student-manager.js (hunk 4 above)
Comment (varsha84) [reply javascript/javascript1/week2/student-manager.js]: I did not thought about it. added one more condition now.

File: javascript/javascript1/week2/student-manager.js (hunk 5 above)
Comment (varsha84) [reply javascript/javascript1/week2/student-manager.js]: I tryed to fix it or I can do something else.

File: javascript/javascript1/week2/student-manager.js (hunk 6 above)
Comment (varsha84) [reply javascript/javascript1/week2/student-manager.js]: done now

File: javascript/javascript1/week2/weather-wear.js (hunk 7 above)
Comment (varsha84) [reply javascript/javascript1/week2/weather-wear.js]: it did not get it. could you explain it?

Comment (jacobemcken) [reply javascript/javascript1/week2/weather-wear.js]: My guess would be:
```js
    ...
    else if(temprature <= 35)
    {
        return "wear beach clothes";
    }
    else
    {
        return" stay at home";
    }
}
```
Will give the same result as:
```js
    ...
    else if(temprature <= 35)
    {
        return "wear beach clothes";
    }

    return" stay at home";
}
```

**Pull request merged**